style.css

```diff
@@ -583,7 +578,211 @@ textarea { resize: none; }
   font-weight: 400;
   font-size: 20px;
   line-height: 32px;
-  color: #42526e;
+}
+
+/* Modal CSS */
+
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.4);
+  z-index: 1;
+  overflow: auto;
+  display: none;
+  justify-content: center;
+  align-items: flex-start;
+}
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 
- -  Please blur the sub background when the popup is open, it will make the popup more like Figma 👍🏿
- - It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 
- - Let's add the popup windows image and  Make sure the max width takes the full width as shown on the Figma design
- - Let's add the missing icons to the  `See live and See source` button ✍🏻 

Your Implementation | Figma Design
:---:|:---:
![Screenshot from 2022-10-19 17-43-46](https://user-images.githubusercontent.com/79058364/196741380-a2fb279f-e850-496b-b31b-ec0642e1ee70.png)  | ![Screenshot from 2022-10-19 17-43-05](https://user-images.githubusercontent.com/79058364/196741368-f7f4bb6a-e49b-498c-ad3f-dc4fcc2ca0fa.png)


- [ ]  Please do the same for the desktop size 👨🏻‍🎨 🎨 

- Actually, Your popup desktop does not really match the figma disign. Please improve your design 🎨 👨🏻‍🎨

Your Implementation | Figma Design
:---:|:---:
![Screenshot from 2022-10-19 17-53-11](https://user-images.githubusercontent.com/79058364/196742519-31a5f96d-5903-4534-9902-55a93356049f.png) | ![Screenshot from 2022-10-19 17-37-43](https://user-images.githubusercontent.com/79058364/196741361-db527420-ffe7-4d5f-8d4e-4a1b84847e6e.png)

Reply:
- [ ] Hello @TayyabKM Kindly make sure to follow the suggested changes by the previous reviewer and implement them because I have noticed that you submitted your project again without making changes according to what you have suggested to change.
 ### EVEN ON MY END, AM STILL GETTING THE SAME RESULT AS THIS ONE.